content/preferences-settings/import.md

```diff
@@ -42,11 +42,11 @@ $(EXIF_SECOND)       seconds the photo was taken (from Exif data)
 $(EXIF_ISO)          ISO value of the photo (from Exif data)
 ```
 
-base directory naming pattern
-: The base directory part of the naming pattern (default `$(PICTURES_FOLDER)/Darktable`).
+base filmroll's directory naming pattern
+: The base filmroll's directory part of the naming pattern (default `$(PICTURES_FOLDER)/Darktable`).
```

Comment:
Hmm I'm not sure I like the wording of this option. TBH I'm not sure the word "filmroll" is really needed here at all. Obviously we can't change the parameter name at this point but I think we can probably leave the description unchanged (just remove the word "filmroll's"?)

Reply:
done